src/chesster.js

```diff
@@ -970,3 +972,87 @@ function(bot, message) {
     }
 });
 
+/* subscriptions */
+
+chesster.hears({
+    middleware: [],
+    patterns: ['^tell'],
+    messageTypes: ['direct_message']
+},
+function(bot, message) {
+    var deferred = Q.defer();
+    bot.startPrivateConversation(message, function (response, convo) {
+        subscription.processTellCommand(chesster.config, message).then(function(response) {
+            convo.say(response);
+            deferred.resolve();
+        }).catch(function(error) {
+            convo.say("I'm sorry, but an error occurred processing this subscription command");
+            deferred.reject(error);
+        });
+    });
+    return deferred.promise;
+});
+
+chesster.hears({
```

Comment:
One thing that might be convenient is to have the subscription command work the same way git does in regards to syntax:

subscription status - Show subscriptions
subscription remove (sub) - Removes a subscription
subscription add (event) - Adds a subscription

Obviously this is just a suggestion, but it might make it more straightforward to add new commands in the future if that's necessary as we'll have a standard model for how the commands are structured.